mitodl/micromasters
# Use the romanized names on the exam card

#### What are the relevant tickets?
closes #2735

#### What's this PR do?

This changes things so that the final exam card will show the romanized version of the user's name, if present, and else will show the first and last names.

#### How should this be manually tested?

Set a first and last name on your profile and visit the dashboard. You should see those values in the final exam card.

Then add non-roman characters (e.g. 🎉) to your first and last names, so that you'll have to fill in the romanized fields. Add new romanized versions and visit the dashboard - you should now see these in the final exam card.
